api/src/permissions/permissions.service.ts

```diff
@@ -42,44 +42,57 @@ export class PermissionSystem {
         return this._id;
     }
 
+    // TODO: Add access map extraction function to extract map to be passed to CMS
+    // /**
+    //  * Extract an access map for the passed group ID's
+    //  * @param groupIds - Group IDs for which the access map should be extracted
+    //  * @returns - Access Map for the passed group IDs
+    //  */
+    // static getAccessMap(groupIds: Array<Uuid>): AccessMap {
+    //     const resultMap = new AccessMap();
+    //     groupIds.forEach((id: Uuid) => {
+    //         const g = groupMap[id];
+    //         if (!g) return;
+
+    //         resultMap.map = { ...resultMap, ...g._groupTypePermissionMap };
+    //     });
+    //     return resultMap;
+    // }
```

Comment:
Not sure if you're immediately working on this in a follow-up PR, otherwise I would suggest to stash this somewhere else and remove the commented code. It's all too easy for dead code like this to continue living in the far future as nobody knows whether we need it or not 😅 So as a general rule I feel it's safer to not commit explicitly commented code.